__tests__/app.test.js

```diff
@@ -53,6 +53,33 @@ describe("GET /api/users", () => {
     });
 })
 
+describe("GET /api/reviews/", () => {
+    test("status code 200, responds with an array of review objects", () => {
+        return request(app)
+            .get('/api/reviews/')
+            .expect(200)
+            .then(({ body }) => {
+                const { reviews } = body;
+                expect( reviews ).toBeInstanceOf(Array);
+                expect(reviews).toHaveLength(13);
+                expect(reviews).toBeSortedBy('created_at', { descending: true });
+                reviews.forEach((review) => {
+                    expect(review).toEqual(
+                        expect.objectContaining({
```

Comment:
Is this strictly testing that a review body has NOT been returned? ojectContaining may give you a false positive here.

Reply:
While I can see in your model you are not asking for the body, could you make the test more robust?

Reply:
I'll give it a try!